types/index.d.ts

```diff
@@ -1,6 +1,6 @@
 import DataTypes = require("./lib/data-types");
 import Deferrable = require("./lib/deferrable");
-import * as Op from "../lib/operators";
+import Op = require('../lib/operators');
```

Comment:
```suggestion
import Op = require("../lib/operators");
```

Reply:
Just to make it consistent with the rest of the requires, but we'll fix it later with the updated eslint config

Reply:
Changed :)